helm/policies-common/templates/Cluster.yaml

```diff
@@ -10,15 +10,15 @@ spec:
         kinds:
         - cluster.x-k8s.io/v1alpha3/Cluster
     context:
-    - name: releases
+    - name: release
       apiCall:
         urlPath: "/apis/release.giantswarm.io/v1alpha1/releases"
-        jmesPath: "items[*]"
+        jmesPath: "items[?metadata.name == 'v{{ `{{` }} request.object.metadata.labels.\"release.giantswarm.io/version\" {{ `}}` }}'] | [0]"
```

Comment:
@giantswarm/team-ludacris I got this change (and others) when I ran `template.sh`. Do you know if the version in the `helm` directory or the version in the `policies` directory is the correct one?

Reply:
That's on me not running it :facepalm:
I will submit a PR tomorrow morning to fix this, so your diff is fine.

Reply:
I also really need to add it to CI with a diff

Reply:
This is in sync now.